components/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/WorkflowImplService.java

```diff
@@ -54,6 +54,14 @@ void addBPSProfile(BPSProfile bpsProfileDTO, int tenantId)
      */
     void removeBPSProfile(String profileName) throws WorkflowImplException;
 
+    /**
+     * Remove all BPS profiles of given tenant id.
+     *
+     * @param tenantId
+     * @throws WorkflowImplException
+     */
+    void removeBPSProfiles(int tenantId) throws WorkflowImplException;
```

Comment:
Should not we use 'default' here?
If not all the classes implementing this interface will have to implement this method.

Reply:
Agree!